File: scenarios/telemetry/dotnet/telemetry_consumer/PositionTelemetryConsumer.cs
using GeoJSON.Text.Geometry;
using MQTTnet.Client;
using MQTTnet.Client.Extensions;

namespace telemetry_consumer
{
    internal class PositionTelemetryConsumer : TelemetryConsumer<Point>
    {
        public PositionTelemetryConsumer(IMqttClient mqttClient)
            : base(mqttClient, new Utf8JsonSerializer(), "vehicles/+/position")
        {
        }
    }
}

File: scenarios/telemetry/ts/telemetryConsumer/src/positionTelemetryConsumer.ts
import {
    MqttClient
} from 'mqtt';
import {
    GeoJsonPoint,
    Utf8JsonSerializer,
    TelemetryConsumer
} from '@mqttapplicationsamples/mqttjsclientextensions';

export class PositionTelemetryConsumer extends TelemetryConsumer<GeoJsonPoint> {
    constructor(mqttClient: MqttClient) {
        super(mqttClient, new Utf8JsonSerializer(), "vehicles/+/position");
    }
}
